File: JavaScript/PollingSystem/Result.js
class Result {
  pollId = 0;
  createdBy = 0;
  option = '';
  userId = 0;
  submittedAt = new Date();

  constructor(pollId, option, userId) {
    this.pollId = pollId;
    this.option = option;
    this.userId = userId;
    this.submittedAt = Date.now();
  }
}

module.exports = {Result};

File: Go/PollingSystem-go/result.go
package main

import "time"

type Result struct {
	PollID      int
	Option      string
	UserID      int
	SubmittedAt int64
}

func NewResult(pollID int, option string, userID int) *Result {
	return &Result{
		PollID:      pollID,
		Option:      option,
		UserID:      userID,
		SubmittedAt: time.Now().UnixMilli(),
	}
}
